src/main/java/eu/deltasource/internship/hotelmanagementsystem/hotel/service/domain/commodities/AbstractCommodity.java

```diff
@@ -0,0 +1,63 @@
+package eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities;
+
+/**
+ * This is an abstract class
+ * which has one private member - inventoryNumber
+ * getters, setters, abstract method prepare() and overridden equals() and hashcode()
+ */
+
+abstract public class AbstractCommodity {
+
+    private static int inventoryNum;
+    protected int inventoryNumber;
+
+    /**
+     * Default constructor
+     * assigns value to the inventory number
+     */
+
+    public AbstractCommodity() {
+        inventoryNumber = ++inventoryNum;
```

Comment:
this is ok for now, but let's name it to a more specific name, maybe GLOBAL_INVENTORY_COUNTER or something like it. Remember the convention for static variable names.